bukkit/src/main/java/net/william278/papiproxybridge/BukkitPAPIProxyBridge.java

```diff
@@ -48,7 +48,9 @@ public void onLoad() {
     public void onEnable() {
         // Register the plugin message channel
         getServer().getMessenger().registerOutgoingPluginChannel(this, getChannel());
+        getServer().getMessenger().registerOutgoingPluginChannel(this, getComponentChannel());
```

Comment:
Now I do wonder, perhaps it'd be easier to just have one channel and include a tiny flag in the message itself (0/1) to determine if it's a component-formatted message? Just to save on some slight user-network overheads for users not using both channels.

Reply:
Is there any overhead using an extra channel? This was fairly easy to introduce and will be entirely backwards compatible from the backend side

Reply:
Fair enough, back compatibility isn’t something I considered